docs/design/admonitions/index.mdx

```diff
@@ -0,0 +1,306 @@
+
+If you are unfamiliar with the Docusaurus admonitions and their apeparance, visit the Docusaurus
+official documentation relating to admonitions by clicking the button below.
+
+<DocusaurusButton label="Docusaurus Documentation: Admonitions" to="https://docusaurus.io/docs/markdown-features/admonitions" />
+
+
+To understand how admonitions work, we need to utilize Inspect Element to find things such
+as HTML class names and CSS values that the admonitions use.
+
+### Using Inspect Element
+
+The best way to see how admonitions work (if you are using Chrome) is to highlight the title
+of the admonition, right click, and select "Inspect". This will bring up Inspect Element.
+Inspect is key to figuring out how to edit the defualt admonitions.
+
+:::tip
+
+If you want to make changes to any element of your website, but find yourself stuck, Inspect
+Element is a powerful tool for making local, temporary changes!
+
+:::
+
+![Use Inspect on the "note" admonition's title](inspect-admonition.png)
+
+From here, we can find the HTML of all the docusaurus page elements. For this guide, we are only
+going to focus on the admonitions.
+
+## Analyzing the HTML (Using Inspect)
+
+When clicking inspect on your site, you may find something like this:
+
+![Inspect Element of a Docusaurus website](inspect-element.png)
+
+Don't be afraid! This is the HTML that Docusaurus generates to make your website!
+If you ckick around in Inspect Element, you will eventually find the elements that
+are the admonitions. We can see that the "note" admonition that we highlighted is
+an HTML `<div>` element, with a couple of classes attatched to it.
+
+![The "note" admonition's HTML code](note-admonition-html.png)
+
+For our purposes, we are intereted in that last class: `alert--secondary`. This is
+the driving class that tells the "note" admonition what colors to use! We can find what
+variables it uses with Inspect Element as well. Find the "Styles" tab in Inspect Element
+and use the seach bar to find the "note" admonition's class we found: `alert--secondary`.
+
+## Analyzing the CSS (Using Inspect)
+
+![The "note" admonition's CSS code](note-css.png)
+
+This is the note admonitions CSS code. This is what tells the note admonition what
+colors to use. We see four arttibutes and their cooresponding variables:
+
+### Note admonition's default CSS
+
+<table>
+    <tr>
+        <th>CSS Attribute</th>
+        <th>Default Value</th>
+    </tr>
+    <tr>
+        <td>`--ifm-alert-background-color`</td>
+        <td>`var(--ifm-color-secondary-contrast-background)`</td>
+    </tr>
+    <tr>
+        <td>`--ifm-alert-background-color-highlight`</td>
+        <td>`rgba(235, 237, 240, 0.15`)</td>
+    </tr>
+    <tr>
+        <td>`--ifm-alert-foreground-color`</td>
+        <td>`var(--ifm-color-secondary-contrast-foreground)`</td>
+    </tr>
+    <tr>
+        <td>`--ifm-alert-border-color`</td>
+        <td>`var(--ifm-color-secondary-dark)`</td>
+    </tr>
+</table>
```

Comment:
I've been using HTML syntax for tables since I found it easier to edit in my IDE and for custom CSS class injections. I'll update it to markdown syntax for this repo.

Reply:
Resolved in https://github.com/DocusaurusCommunity/website/pull/32/commits/d6971301b03587f28afd339fb0ed4f5ab20a6b22

Reply:
Aha! Yup, if I had to inject an inline style specific to a table or if something could not be done with the Markdown syntax, I would also consider writing it down in HTML.

Thanks for the update!